src/main/java/com/connexta/search/index/Index.java

```diff
@@ -26,4 +26,7 @@
 
   @Indexed(name = SolrConfiguration.CONTENTS_ATTRIBUTE_NAME, type = "string")
   private String contents;
+
+  @Indexed(name = SolrConfiguration.MEDIA_TYPE_ATTRIBUTE_NAME, type = "string")
```

Comment:
@peterhuffer The Sprint Data library handles the Solr schema. The @Indexed annotation is part of that library.

Reply:
I just started the stack and hit the solr admin UI and looked at the `searchTerms` collections schema. These annotations do not create the fields. It looks like we are using the default solr schema, which is relying on dynamic fields. At some point we probably don't want dynamic fields and want to define our own unless we expect to index and search on arbitrary fields.

Reply:
The default solr schema also has some comments about removing some sections for performance reasons if they are not needed.

Reply:
Yeah. This is a known issue. There are currently issues when we query before anything is stored.